routes/BookingRouter.js

```diff
@@ -42,9 +43,10 @@ bookingRouter.post("/booking", authVerify, async (req, res) => {
         const BOOKING_NOTIFICATION_ENDPOINT = process.env.BOOKING_NOTIFICATION_ENDPOINT
         const username = user.username
         const email = user.email
-
-        await sendEmail(username, email, req.body.space_name, req.body.check_in, req.body.check_out, BOOKING_NOTIFICATION_ENDPOINT)
-        console.log("Booking successful email sent to" + username)
+        const check_in =  moment(req.body.check_in).format("dddd, MMMM Do YYYY, h:mm:ss a")
```

Comment:
put the format in a file @Amotul-raheem and call it here. it's repeated below as well.